tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_browser_Browser.java

```diff
@@ -518,6 +518,18 @@ public void changing(LocationEvent event) {
 	for (int i = 0; i < 100; i++) browser.removeLocationListener(listener);
 }
 
+
+
+@Test
+public void test_isLocationForCustomText() {
+	AtomicBoolean changingFired = new AtomicBoolean(false);
+	browser.addLocationListener(changingAdapter(e -> changingFired.set(true)));
+	shell.open();
+	browser.setText("Hello world");
+	waitForPassCondition(changingFired::get);
+	assertTrue("LocationListener.changing() event was never fired", browser.isLocationForCustomText(browser.getUrl()));
```

Comment:
The assertion message is unrelated to what the assertion actually checks. I propose to make two assertions out of this:
1. one assertion checking that `LocationListener.changing()` has fired
2. one assertion checking the return value of the new method.